glommio/src/executor/mod.rs

```diff
@@ -2021,7 +2062,13 @@ mod test {
     }
 
     #[test]
+    #[allow(unused_must_use)]
     fn bind_to_cpu_set_range() {
+        tracing_subscriber::fmt::fmt()
```

Comment:
shouldn't init be something for the application to do ?

This is fine in some of our tests, I guess, but this is a particularly interesting place for it. Why here?

Reply:
Yeah this is just testing that, but it's ok I moved it in my next patch I just uploaded.